cinema/tests/test_movie_api.py

```diff
@@ -117,9 +113,7 @@ def test_filter_movies_by_actors(self):
 
         movie3 = sample_movie(title="Movie without actors")
 
-        res = self.client.get(
-            MOVIE_URL, {"actors": f"{actor1.id},{actor2.id}"}
-        )
+        res = self.client.get(MOVIE_URL, {"actors": f"{actor1.id},{actor2.id}"})
```

Comment:
Similar to the genre filtering, ensure that the API supports filtering movies by multiple actor IDs. The test assumes this functionality, so confirm that the API is implemented to handle such queries.